CHANGELOG.md

```diff
@@ -99,30 +112,44 @@ bring in a verification key caching fix. [[GH-11784](https://github.com/hashicor
 * core: Fix storage entry leak when revoking leases created with non-orphan batch tokens. [[GH-11377](https://github.com/hashicorp/vault/pull/11377)]
 * core: Fixed double counting of http requests after operator stepdown [[GH-11970](https://github.com/hashicorp/vault/pull/11970)]
 * core: correct logic for renewal of leases nearing their expiration time. [[GH-11650](https://github.com/hashicorp/vault/pull/11650)]
+* core: initialized unlicensed raft nodes were starting instead of failing with an error. [[GH-1989](https://github.com/hashicorp/vault/pull/1989)]
```

Comment:
I think we can remove this one.  I was wrong to add a CL, I think I did so because it was post RC1 and I was under the impression we would track 1.8 feature bugfixes that way.  I've since learned that's not the policy.

Reply:
Updated